client/src/components/caseStudy/CaseStudyWconnection.tsx

```diff
@@ -4,13 +4,59 @@ import Button from "@mui/material/Button";
 import "../img/normal_search.svg";
 import { PieChart } from "react-minimal-pie-chart";
 import Table from './Table';
-import { useNavigate } from 'react-router-dom';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { Case } from "../../model/CaseStudy";
+import axios from "axios";
+import { Dialog, DialogActions, DialogContent, DialogTitle } from "@mui/material";
+import FileDownloadIcon from "@mui/icons-material/FileDownload";
 
-interface Props {}
+const CaseStudyWTconnection = () => {
```

Comment:
le nom ici dit WT mais le nom du fichier dit juste W, c'est normal?

Reply:
No clue, j’ai même pas remarqué haha. Y’a deux fichiers qui ont simili le même nom mais y’en a un qui est pas utilisé, maybe take a look to see if we can delete one and change the other’s name @EdwardHealy00

Reply:
Les deux fichiers semblent servir de template hardcodé pour la consultation d'une étude de cas. Selon l'historique de commits, j'en déduis que "WTconnection" signifie "without connection" puisque celui-ci n'utilise pas les données de la db dans son affichage contrairement à "Wconnection" (with connection). Le nom de variable qui ne correspond pas à celui du fichier est fort probablement une erreur de copy-paste. Je vais commit la modif et merge la PR.